.travis.yml

```diff
@@ -58,6 +69,11 @@ install:
   - sudo service postgresql stop
 
 before_script:
+  - cd src/portal
+  - npm install -g -q --no-progress angular-cli
+  - npm install -g -q --no-progress karma
+  - npm install -q --no-progress
+  - npm run build_lib && npm run link_lib && cd ../..
```

Comment:
I think we should add a new script in package.json to do the "build all" job , to finish everything. The benifit for this is to decouple the changes from ui to CI pipe.